sec1_intro_and_setup/test_app/app/views/layouts/mailer.text.erb

```diff
@@ -0,0 +1 @@
+<%= yield %>
```

Comment:
What is this `yield`?

Reply:
`yield` is used to add the appropriate view for the page the user is looking at.

Reply:
How?

Reply:
The `yield` here is occupying a place where you will write code in the future. The page will render the layout file until it hits the `yield` statement, then it stops and drops in the template file(the one you will write your code). After finishing the template, the page keeps going to render everything after the `yield` statement in the layout.

Reply:
you still didn't answer the question.

Let me give a scenario:

aaa.html.erb content:

```
<h1>aaa here</h1>
<%= yield %>
<h1>aaa here end</h1>
```

bbb.html.erb content:
```
<h1>test bbb</h1>
```

How can I get bbb to be "yield" into aaa.html.erb based on what you said?

Reply:
`<%= yield :bbb.html.erb%>`

Reply:
:( 
lets move on